linera-rpc/src/grpc/client.rs

```diff
@@ -94,43 +94,77 @@ impl GrpcClient {
             }
```

Comment:
Should the log be an `error` in the last case?

Reply:
Probably in both "unexpected" cases?